helpers/datasources.ts

```diff
@@ -11,6 +11,21 @@ export const EXAMPLE_FILE: TemplateDataSource = {
   },
 };
 
+export const EXAMPLE_10K_SEC_FILES: TemplateDataSource[] = [
+  {
+    type: "file",
+    config: {
+      path: "https://s2.q4cdn.com/470004039/files/doc_earnings/2023/q4/filing/_10-K-Q4-2023-As-Filed.pdf",
+    },
+  },
+  {
+    type: "file",
+    config: {
+      path: "https://ir.tesla.com/_flysystem/s3/sec/000162828024002390/tsla-20231231-gen.pdf",
```

Comment:
could add more files if download time is still resonable

Reply:
I think two files is enough for the current use case. Download time is not an big issue, but with multiple files it will takes time for indexing, especially with LlamaCloud - I often run into maxAttemp issue with more than 2 files.